.github/workflows/Pkg EdFi.DmsApi.Sdkyml

```diff
@@ -3,25 +3,28 @@
 # The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
 # See the LICENSE and NOTICES files in the project root for more information.
 
-name: Data Management System SDK Generation
+name: Pkg EdFi.DmsApi.Sdk
```

Comment:
This workflow will get executed only when i get merged this PR .  Since it's new workflow